fuzz.go

```diff
@@ -179,6 +192,12 @@ const (
 )
 
 func (f *Fuzzer) doFuzz(v reflect.Value, flags uint64) {
+	if f.curDepth >= f.maxDepth {
+		return
+	}
+	f.curDepth++
```

Comment:
Do we care?  The alternative is to carry around a context object up and down the call stack?